posthog/hogql_queries/insights/trends/query_builder.py

```diff
@@ -300,6 +300,34 @@ def _inner_select_query(self, inner_query: ast.SelectQuery | ast.SelectUnionQuer
             ),
         )
 
+        if (
+            self.query.trendsFilter is not None
+            and self.query.trendsFilter.smoothing_intervals is not None
+            and self.query.trendsFilter.smoothing_intervals > 1
+        ):
+            smoothing_count = ast.Alias(
```

Comment:
I would love if we switched the vague "smoothing" terminology to "rolling average". This was a UI issue we saw with some of the users we opted into this beta feature. In the code this is not so much as UX issue, but still worth having a single accurate name (except `smoothing_intervals` I guess, since that's an existing schema)

Reply:
Nice yeah, have updated this